File: Python-Assignment/chapterThree/Question10.java
import java.util.Scanner;

public class Question10{
   public static void main(String []args){

	Scanner input = new Scanner (System.in);

	System.out.print("Enter investment amount: ");
	int investmentAmount = input.nextInt();

	System.out.print("Enter annual return rate: ");
	int annualReturnRate = input.nextInt();

	double percentageAnnualReturn = annualReturnRate / 100.0;

	double percentageAnnualReturnPlusOne = 1 + percentageAnnualReturn;

	for (int noOfYrs = 1; noOfYrs <= 30; noOfYrs++){
		double percentageAnnualReturnRaisedToPowerNoOfYrs = Math.pow(percentageAnnualReturnPlusOne, noOfYrs);

		double result = investmentAmount * percentageAnnualReturnRaisedToPowerNoOfYrs;

		System.out.printf("The investment amount after %d years: %.2f%n", noOfYrs, result);
	}
}

}

File: Python-Assignment/chapterThree/question10.py
investment_amount = int(input("Enter investment amount: "))
annual_return = int(input("Enter annual return rate: "))
percentage_annual_return = annual_return / 100
added_percentage_annual_return = percentage_annual_return + 1

for no_of_years in range(1, 31):
	added_annual_return_power = pow(added_percentage_annual_return, no_of_years)
	product = investment_amount * added_annual_return_power
	print ("The investment amount after ", no_of_years, " years: ", product)